parser/src/grammar.lalrpop

```diff
@@ -167,6 +168,12 @@ AppType = {
 Type = {
     AppType,
 
+    "forall" <args: Ident+> "." <ty: Type> =>
+        Type::forall(args.into_iter()
+                         .map(|id| Generic { id: id, kind: Kind::variable(0) })
```

Comment:
Maybe, but it this variable will never get read from in any case so I don't think its necessary (for types `Hole` is at least used other than a dumy value).